### DISCUSSION TITLE
What exactly is this thing supposed to do?

### DISCUSSION THREAD

**Discussion by scottonanski** [apollographql/apollo-client-devtools#1252] [Q&A]
I've opened it, but I have no idea what I'm looking at. How is this supposed to help as opposed to regular dev tools?

**Comment by jerelmiller** [comment]
Hey @scottonanski 👋 

I'd recommend taking a look at the feature section in the [README](https://github.com/apollographql/apollo-client-devtools?tab=readme-ov-file#features) to give you a high level overview of what these devtools provide when using with Apollo Client.

**Reply by jerelmiller** [reply]
We also have a section in our docs about the devtools that you might find useful 🙂 https://www.apollographql.com/docs/react/development-testing/developer-tooling/#apollo-client-devtools

**Comment by scottonanski** [comment]
I looked at those. They presuppose prior knowledge. I was inspecting something in the browser and noticed a link to Apollo. Still don't get it.

At any rate, you might want to take a look how how well documentation used to be written circa 2010, and maybe follow those examples. 99% of the documentation written today is absolute trash.

**Reply by jerelmiller** [reply]
> I looked at those. They presuppose prior knowledge.

I would agree that they imply that you've got some familiarity with Apollo Client and GraphQL. This is our intended audience at the moment. We've got a wealth of docs about [Apollo Client itself](https://www.apollographql.com/docs/react/) and think these serve to explain what Apollo Client is and how it works. 

We haven't and don't plan to pour much time into teaching GraphQL/Apollo Client concepts in the devtools documentation since we feel that is reserved for the other parts of our documentation that can do it much better. The devtools are meant as a supplementary tool to Apollo Client. You likely won't find much value with Apollo devtools unless you're already using and developing with Apollo Client.

Perhaps I should ask, are there _specific_ questions you have about the devtools? Are you familiar with Apollo Client and/or have you developed much with it? I'd be happy to answer any questions you have about what parts of Apollo Client integrate with the various parts of the devtools.

> Still don't get it.

I'd like to ask: how would you change the documentation here to be more useful? It's difficult for me to understand what you're struggling with. I'd be happy to add some clarification in the devtools documentation that might be too vague.

> you might want to take a look how how well documentation used to be written circa 2010

Do you have some examples you can share? This seems pretty vague to me so I'd love to understand what aspects of documentation you have in mind here. Anything you can provide would be useful.

> 99% of the documentation written today is absolute trash.

I don't know if this is a general statement or directed at us, but I'd appreciate constructive criticism instead. There's always room for improvements and sometimes what makes sense to us can be vague to others. Additionally, we are an open source library, so if you have ideas on how to clarify, we accept PRs.

**Comment by jerelmiller** [comment]
Hey @scottonanski 👋 

Since we haven't had a reply, I'll assume there is enough information here for you. As such I'm going to go ahead and close this discussion. Please let us know if you have additional questions and we'd be happy to reopen this discussion. Thanks!
